dev/doc/release-process.md

```diff
@@ -80,6 +80,10 @@
 ## Before the beta release date ##
 
 - [ ] Ensure the Credits chapter has been updated.
+- [ ] Prepare the release notes (see e.g., [#10833](https://github.com/coq/coq/pull/10833)):
+  in a PR against the `master` branch, move the contents from all files of
+  `doc/changelog/` into the manual `doc/sphinx/changes.rst`.
+  Merge that PR into the `master` branch and backport it to the version branch.
```

Comment:
Just to clarify "the contents from all files" should be understood as "all files appearing in the release branch".
You can also add a TODO write a script for this as I indeed intend to automate that.